example/otel-collector/README.md

```diff
@@ -13,165 +13,17 @@ App + SDK ---> OpenTelemetry Collector ---|
 
 # Prerequisites
 
-You will need access to a Kubernetes cluster for this demo. We use a local
-instance of [microk8s](https://microk8s.io/), but please feel free to pick
-your favorite. If you do decide to use microk8s, please ensure that dns
-and storage addons are enabled
+You will need [Docker Compose V2](https://docs.docker.com/compose/) installed for this demo.
 
-```bash
-microk8s enable dns storage
-```
-
-For simplicity, the demo application is not part of the k8s cluster, and will
-access the OpenTelemetry Collector through a NodePort on the cluster. Note that
-the NodePort opened by this demo is not secured.
-
-Ideally you'd want to either have your application running as part of the
-kubernetes cluster, or use a secured connection (NodePort/LoadBalancer with TLS
-or an ingress extension).
-
-If not using microk8s, ensure that cert-manager is installed by following [the
-instructions here](https://cert-manager.io/docs/installation/).
-
-# Deploying to Kubernetes
-
-All the necessary Kubernetes deployment files are available in this demo, in the
-[k8s](./k8s) folder. For your convenience, we assembled a [makefile](./Makefile)
-with deployment commands (see below). For those with subtly different systems,
-you are, of course, welcome to poke inside the Makefile and run the commands
-manually. If you use microk8s and alias `microk8s kubectl` to `kubectl`, the
-Makefile will not recognize the alias, and so the commands will have to be run
-manually.
-
-## Setting up the Prometheus operator
-
-If you're using microk8s like us, simply do
-
-```bash
-microk8s enable prometheus
-```
-
-and you're good to go. Move on to [Using the makefile](#using-the-makefile).
-
-Otherwise, obtain a copy of the Prometheus Operator stack from
-[prometheus-operator](https://github.com/prometheus-operator/kube-prometheus):
-
-```bash
-git clone https://github.com/prometheus-operator/kube-prometheus.git
-cd kube-prometheus
-kubectl create -f manifests/setup
-
-# wait for namespaces and CRDs to become available, then
-kubectl create -f manifests/
-```
-
-And to tear down the stack when you're finished:
-
-```bash
-kubectl delete --ignore-not-found=true -f manifests/ -f manifests/setup
-```
-
-## Using the makefile
-
-Next, we can deploy our Jaeger instance, Prometheus monitor, and Collector
-using the [makefile](./Makefile).
-
-```bash
-# Create the namespace
-make namespace-k8s
-
-# Deploy Jaeger operator
-make jaeger-operator-k8s
-
-# After the operator is deployed, create the Jaeger instance
-make jaeger-k8s
+# Deploying to docker compose
```

Comment:
Could we still have a section with the command to deploy to k8s?

Reply:
Do you mean we should keep the example for k8s?

Reply:
Wouldn't that be an "otel-collector example for k8s" rather than an "otel-go SDK example using the collector"?
For the former, I believe that example would be better suited within the collector itself.